ci/travis/build-autoscaler-images.sh

```diff
@@ -9,40 +9,68 @@ SCRIPT_DIR=$(cd "$(dirname "${BASH_SOURCE:-$0}")"; pwd)
 ROOT_DIR=$(cd "$SCRIPT_DIR"/../../; pwd)
 DOCKER_USERNAME="raytravisbot"
 
+docker_push() {
+    if [[ "$TRAVIS_PULL_REQUEST" == "false" ]]; then
+        docker push $@
+    else
+        echo "Skipping docker push because it's in PR environment."
+    fi
+}
+
 # We will only build and push when we are building branch build.
-if [[ "$TRAVIS" == "true" && "$TRAVIS_PULL_REQUEST" == "false" ]]; then
-    echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
+if [[ "$TRAVIS" == "true" ]]; then
 
-    wheel="$(basename "$ROOT_DIR"/.whl/*cp36m-manylinux*)"
+    if [[ "$TRAVIS_PULL_REQUEST" == "false" ]]; then
+        echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
+    else
+        if [[ "$RAY_CI_DOCKER_AFFECTED" == "0" ]]; then
+            echo "Skipping docker build in PR build because dockerfile didn't change."
+            exit 0
+        fi
+    fi
+
+    wheel="$(basename "$ROOT_DIR"/.whl/*cp37m-manylinux*)"
     commit_sha=$(echo "$TRAVIS_COMMIT" | head -c 6)
     cp -r "$ROOT_DIR"/.whl "$ROOT_DIR"/docker/autoscaler/.whl
+    cp $ROOT_DIR/python/requirements.txt $ROOT_DIR/docker/autoscaler/requirements.txt
+    cp $ROOT_DIR/python/requirements_autoscaler.txt $ROOT_DIR/docker/autoscaler/requirements_autoscaler.txt
 
-    docker build -q -t rayproject/base-deps docker/base-deps
+    docker build -t rayproject/base-deps docker/base-deps
```

Comment:
there's probably no reason to tag it on the local docker-daemon as `docker/base-deps` -- unless we're using that exact image name somewhere (then, why not make it `rayproject/base-deps`?) and we certainly can't push to it

Reply:
I think the tag is `rayproject/base-deps`, the directory is just `docker/base-deps`

Reply:
Derp. yep